modules/snmptrapd-parser/snmptrapd-parser.c

```diff
@@ -195,10 +199,20 @@ snmptrapd_parser_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *
   log_msg_set_value(nv_context.msg, LM_V_PROGRAM, "snmptrapd", -1);
 
   if (!snmptrapd_header_parser_parse(&nv_context, &input, &input_len))
-    return FALSE;
+    {
+      msg_debug("  snmptrapd-parser failed",
+                evt_tag_str ("error", "can not parse header"),
+                evt_tag_str ("input", input));
+      return FALSE;
+    };
 
   if (!_parse_varbindlist(&nv_context, &input, &input_len))
-    return FALSE;
+    {
+      msg_debug("  snmptrapd-parser failed",
+                evt_tag_str ("error", "can not varbindlist"),
+                evt_tag_str ("input", input));
```

Comment:
can not parse name-value pairs in the input

Reply:
Thanks, fixed.